source/source-connector/fundamentals/json-formatters.txt

```diff
@@ -0,0 +1,196 @@
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide, you can learn how to specify built-in JSON formatter
+classes to use in a{+source-connector+}.
+
+JSON Formatters
+---------------
+
+**JSON formatters** specify how JSON data appears. They provide instructions
+for how the connector represents different types in output.
+
+Built-In Formatters
+-------------------
+
+The following table describes the formatter classes available in the
+source connector:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Class
+     - Description
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.DefaultJson``
+     - The legacy strict JSON formatter.
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.ExtendedJson``
+     - The fully type-safe extended JSON formatter.
+
+   * - ``com.mongodb.kafka.connect.source.json.formatter.SimplifiedJson``
+     - The simplified JSON formatter. This formatter class represents
+       ``ObjectId``, ``Decimal``, ``Date``, and ``Binary`` values as
+       strings.
+
+Examples
+--------
+
+The following table describes the fields of the sample document that the
+examples in this guide use to demonstrate each output format. The
+columns describe the field name, value, and type for each field or
+nested field.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 45 30
+
+   * - Name
+     - Value
+     - Type
+
+   * - ``_id``
+     - .. code-block:: sh
+          :copyable: false
+          
+          ObjectId('5f15aab12435743f9bd126a4')
+
+       .. code-block:: sh
+          :copyable: false
+         
+          "$oid": "5f15aab12435743f9bd126a4"
+
+     - ``ObjectID`` (``$oid``)
+
+   * - ``w``
+     - .. code-block:: sh
+          :copyable: false
+      
+          [ Decimal128('12345.6789'), 23.53 ]
+
+     - | Array of:
+       | - ``Decimal128`` (``$numberDecimal``)
+       | - ``Double`` (``$numberDouble``)
+
+   * - ``x``
+     - .. code-block:: sh
+          :copyable: false
+      
+          BinData(0, 'SSBsb3ZlIGZvcm1hdHRpbmch')
+
+       .. code-block:: sh
+          :copyable: false
+         
+          "$binary": {
+             "base64": "SSBsb3ZlIGZvcm1hdHRpbmch",
+             "subType": "00"
+          }
+
+     - ``Binary`` (``$binary``)
+
+   * - ``y``
+     - .. code-block:: sh
+          :copyable: false
+         
+          new Date('2023-05-11T08:27:07.000Z')
+
+       .. code-block:: sh
+          :copyable: false
+         
+          "$date": "1683793627000"
+
+     - ``Date`` (``$date``)
+
+   * - ``z``
+     - .. code-block:: sh
+          :copyable: false
+         
+          { a: false, b: 87, c: "hello world" }
+          
+     - | Document with fields:
+       | - ``a``: ``Boolean``
+       | - ``b``: ``Int32`` (``$numberInt``)
+       | - ``c``: ``String``
+
+Default JSON Formatter
+~~~~~~~~~~~~~~~~~~~~~~
+
+In your source connector configuration properties, set the following
```

Comment:
[suggestion] 

Minor rewording suggested to improve readability. I find "source connector configuration properties" difficult to parse/slows down my absorption of the material.

```suggestion
In the configuration properties for your source connector, set the following
```